omod/src/main/java/org/openmrs/module/queue/web/resources/QueueEntrySubResource.java

```diff
@@ -53,7 +53,7 @@ public void setParent(QueueEntry queueEntry, Queue queue) {
 	
 	@Override
 	public PageableResult doGetAll(Queue queue, RequestContext requestContext) throws ResponseException {
-		Collection<QueueEntry> queueEntries = queue.getQueueEntries();
+		Collection<QueueEntry> queueEntries = queue.getActiveQueueEntries();
```

Comment:
Same comment here as above.  This change is needed for this to replicate the existing functionality, but is this what we want?  Should the "getAll" resource limit itself to only active queue entries?  Or should getAll really ...get all.  And then we can limit this to only active entries by passing in a filter or search parameter.

Reply:
Do we have a use case for that already? 
Yeah `getAll` should really get all queue entries and probably add more search parameters through search handlers.

Reply:
I don't have an explicit use case yet.  It was more just a matter of wanting to ensure our API was flexible enough to handle these use cases if/when they come up and also that it makes intuitive sense and matches what users would expect.  Once we establish that this endpoint only returns active queue entries, if a use case _does_ come up where we want _all_ queue entries regardless of active or not, how would we handle that?  Would we have to pass in a parameter saying "includeInactive=true" for example?  Not the end of the world, just putting it out there for consideration as to how we'd want/expect our REST apis to work, and if there are existing conventions in OpenMRS that we honor those.